src/components/views/messages/MAudioBody.js

```diff
@@ -96,9 +96,13 @@ export default class MAudioBody extends React.Component {
 
         const contentUrl = this._getContentUrl();
 
+        const shouldAutoPlay = MatrixClientPeg.get().credentials.userId !==
+          this.props.mxEvent.sender.userId &&
+          content.autoplay;
+
         return (
             <span className="mx_MAudioBody">
-                <audio src={contentUrl} controls />
+                <audio src={contentUrl} controls autoPlay={shouldAutoPlay} />
```

Comment:
>This would also presumably play all files as you scrolled past them in the backlog

Wouldn't this *also* play sounds as they get rendered. So when I log in, or switch rooms? That sounds quite terrifying.